packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.classNames.ts

```diff
@@ -0,0 +1,33 @@
+import { memoizeFunction } from '../../../Utilities';
+import { mergeStyles } from '../../../Styling';
+import { IButtonStyles } from '../Button.Props';
+
+export interface ISplitButtonClassNames {
+  root?: string;
+  icon?: string;
+  flexContainer?: string;
+}
+
+export const getClassNames = memoizeFunction((
+  styles: IButtonStyles,
+  disabled: boolean,
+  expanded: boolean,
+  checked: boolean,
+): ISplitButtonClassNames => {
+  return {
+    root: mergeStyles(
+      styles.splitButtonMenuButton,
+      expanded && [
```

Comment:
I'd leave the array off for legibility if you don't need it, personal preference though.